projects/js-packages/publicize-components/src/components/form/unsupported-connections-notice.tsx

```diff
@@ -4,34 +4,30 @@ import { __ } from '@wordpress/i18n';
 import { usePublicizeConfig } from '../../..';
 import useSocialMediaConnections from '../../hooks/use-social-media-connections';
 import Notice from '../notice';
-import { useService } from '../services/use-service';
 
 export const UnsupportedConnectionsNotice: React.FC = () => {
 	const { connections } = useSocialMediaConnections();
-
 	const { connectionsPageUrl } = usePublicizeConfig();
 
-	const getServices = useService();
-
-	const unsupportedConnections = connections.filter(
-		connection =>
-			// If getServices returns falsy, it means the service is unsupported.
-			! getServices( connection.service_name )
+	const hasTwitterConnection = connections.some(
+		( { service_name } ) => 'twitter' === service_name
 	);
 
+	if ( ! hasTwitterConnection ) {
+		return null;
+	}
```

Comment:
Not that it matters too much, but we were using an `unsupported` status so that we could control if twitter was available or not: fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Syvo%2Srkgreany%2Qpbaarpgvbaf.cuc%3Se%3Q8455q799%23307%2Q323-og

It seems like as we aren't displaying it in the connections list anymore then we were treating the service as unsupported if it wasn't found.

We should probably start filtering out, and removing, these connections on the WPCOM side, in the same way that we did with Google+

Reply:
I have created a task for this - https://github.com/Automattic/jetpack-reach/issues/860